CONTRIBUTING.md

```diff
@@ -1,25 +1,61 @@
 # Contributing
 
-### 1. Setting up the linting pre-commit hook
+To contribute to the FE, open a pull request on [LANE_frontend](https://github.com/dougUCN/LANE_frontend)
+
+To contribute to the BE, open a pull request on [LANE_backend](https://github.com/dougUCN/LANE_backend)
+
+### Working with FE and BE as submodules
+
+A submodule is essentially a reference to a specific commit in a branch in another repo. In our case we have `client` point to the main branch of [LANE_frontend](https://github.com/dougUCN/LANE_frontend), and we have `server` point to the main branch of [LANE_backend](https://github.com/dougUCN/LANE_backend).
+
+If you cloned the repository as instructed in the readme with the `--recurse-submodules` flag, then the FE and BE submodules will be populated with files. When you are in `client`, you will be treated as if you are working directly in the FE repo. For example, `git branch` will display only branches in the FE repo, and `git push` will push staged files to the FE repo. Conversely, when in `server`, you will be treated as if you are directly working in the BE repo.
+
+Now, as mentioned, a submodule is only a reference to a specific commit. To update the main repo, create a new branch and run `git status` to see whether a submodule has new commits. Then run `git add... git commit ... git push` as usual.
+
+If `git status` is not showing changes, `git submodule update --remote --merge` will also update the submodule.
+
+If you cloned the repository without the `--recurse-submodules` flag, then the FE and BE submodule folders will not be populated with files.
```

Comment:
When will repositories be cloned without this flag? If never then it should be mentioned all in the above paragraph.

Reply:
Say that a contributor desires to work only on one repo, or has the FE/BE repos cloned onto separate partitions. If they want to then open a PR on the main LANE repo they can then clone the main repo without the `--recurse-submodules` flag